### PR TITLE
Update to latest version

### PR DESCRIPTION
**Pull request by ccordoba12**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by ccordoba12** [comment]
@conda-forge-admin, please rerender

**Comment by ccordoba12** [comment]
@ocefpaf, could you help me to merge this PR? We need this package for Spyder, thanks!

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,24 +1,22 @@
...
   url: https://pypi.io/packages/source/d/diff-match-patch/diff-match-patch-{{ version }}.tar.gz
-  sha256: 9dba5611fbf27893347349fd51cc1911cb403682a7163373adacc565d11e2e4c
+  sha256: a809a996d0f09b9bbd59e9bbd0b71eed8c807922512910e05cbd3f9480712ddb
 
 build:
   noarch: python
-  number: 1
-  script: python -m pip install --no-deps --ignore-installed .
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (ocefpaf) [inline recipe/meta.yaml:14]: See https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L33

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,24 +1,22 @@
-{% set version = "20121119" %}
+{% set version = "20181111" %}
 
 package:
   name: diff-match-patch
   version: {{ version }}
 
 source:
-  fn: diff-match-patch-{{ version }}.tar.gz
```
Comment (ocefpaf) [inline recipe/meta.yaml:8]: See https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L13

File: recipe/meta.yaml [hunk 3]
```diff
@@ -28,10 +26,11 @@ test:
 
 about:
   home: http://code.google.com/p/google-diff-match-patch/
-  license: Apache 2.0
-  license_file: LICENCE
-  summary: 'The Diff Match and Patch libraries offer robust algorithms to perform the operations required for synchronizing plain text.'
+  license: Apache-2.0
+  license_file: LICENSE
+  summary: 'Diff Match Patch is a high-performance library in multiple languages that manipulates plain text'
```
Comment (ocefpaf) [inline recipe/meta.yaml:31]: That was too long and the one on their GH page is virtually the same but shorter.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -1,24 +1,22 @@
...
-  number: 1
-  script: python -m pip install --no-deps --ignore-installed .
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
 
 requirements:
   host:
-    - pip
     - python
-    - setuptools
```
Comment (ocefpaf) [inline recipe/meta.yaml:21]: No need when using `pip`.

**Review by ocefpaf — APPROVED** [review]

**Comment by ccordoba12** [comment]
Thanks!

**Pull request merged**